src/test/java/com/google/cloud/spanner/jdbc/ITAbstractJdbcTest.java

```diff
@@ -76,10 +76,6 @@ protected static boolean hasValidKeyFile() {
     return getKeyFile() != null && Files.exists(Paths.get(getKeyFile()));
   }
 
-  protected static IntegrationTestEnv getTestEnv() {
```

Comment:
This is a breaking change, no?

Reply:
Yes, I guess it is if someone also imports the test library and extends this. I've re-added this method.